auth/service.go

```diff
@@ -633,7 +633,7 @@ func (svc service) UpdateDomain(ctx context.Context, token, id string, d DomainR
 		ObjectType:  DomainType,
 		Permission:  EditPermission,
 	}); err != nil {
-		return Domain{}, errors.Wrap(svcerr.ErrAuthorization, err)
+		return Domain{}, err
```

Comment:
`svc.Authorize`  this function belongs to Auth service and It is exported function, 
So this function will return error with svcerr wrapped.   
Reference: https://github.com/WashingtonKK/magistrala/blob/81328c217c29766674966f63cb4b834fed586294/auth/service.go#L181-L182     
So here there is no need to warp again here `svcerr`